<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="7" total="9" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-03-11T10:10:24 IST" name="Default suite" finished-at="2023-03-11T10:13:04 IST" duration-ms="159824">
    <groups>
    </groups>
    <test started-at="2023-03-11T10:10:24 IST" name="Default test" finished-at="2023-03-11T10:13:04 IST" duration-ms="159824">
      <class name="com.dinamic.v3.test.tescases.OrdersTestCase">
        <test-method is-config="true" signature="browserConfiguration()[pri:0, instance:com.dinamic.v3.test.tescases.OrdersTestCase@c88a337]" started-at="2023-03-11T10:10:24 IST" name="browserConfiguration" finished-at="2023-03-11T10:10:29 IST" duration-ms="4923" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- browserConfiguration -->
        <test-method signature="login()[pri:1, instance:com.dinamic.v3.test.tescases.OrdersTestCase@c88a337]" started-at="2023-03-11T10:10:29 IST" name="login" finished-at="2023-03-11T10:10:34 IST" duration-ms="5301" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="ordersEntry()[pri:2, instance:com.dinamic.v3.test.tescases.OrdersTestCase@c88a337]" started-at="2023-03-11T10:10:34 IST" name="ordersEntry" finished-at="2023-03-11T10:13:03 IST" duration-ms="148994" status="FAIL">
          <exception class="java.lang.IndexOutOfBoundsException">
            <message>
              <![CDATA[Index 0 out of bounds for length 0]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
at java.base/java.util.Objects.checkIndex(Objects.java:359)
at java.base/java.util.ArrayList.get(ArrayList.java:427)
at com.dinamic.v3.test.pages.OMSPage.viewButtonOrdering(OMSPage.java:286)
at com.dinamic.v3.test.tescases.OrdersTestCase.ordersEntry(OrdersTestCase.java:60)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IndexOutOfBoundsException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ordersEntry -->
        <test-method is-config="true" signature="close()[pri:0, instance:com.dinamic.v3.test.tescases.OrdersTestCase@c88a337]" started-at="2023-03-11T10:13:03 IST" name="close" finished-at="2023-03-11T10:13:04 IST" duration-ms="515" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- com.dinamic.v3.test.tescases.OrdersTestCase -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
